### ISSUE TITLE
How to highlight package for Nova 4 

### ISSUE THREAD

**Issue by scramatte** [tighten/novapackages#323]
Hello,

How can I highlight my packages for Nova 4?

regards

**Comment by RhysLees** [comment]
Hi, I contributed that feature!

You simply need to require a nova 4 version in your package

for example in `composer.json`

```diff
{
  "name": "ideatocode/nova-tooltip-field",
  "description": "A Laravel Nova field to show additional data but not clutter the table.",
  "keywords": [
    "laravel",
    "nova"
  ],
  "license": "MIT",
  "require": {
    "php": "^7.3|^8.0",
+    "laravel/nova": "^4.0"
  },
}
```

**Comment by johnpaulmedina** [comment]
So I had a nova 4 requirement on require-dev as the Nova documentation indicates. Once I submitted the package it did not tag it as Nova4 support and after seeing this issue I bumped the patch version and switched the requirements but it still has not tagged it as Nova4.

I would recommend looking at the dev-requirements as well.

As well as on package version change trigger the support tag check once again.

**Comment by wdelfuego** [comment]
Same here, just submitted (and then re-submitted to be sure) my new package, it has the laravel/nova requirement in composer.json, but no Nova 4 support tag is shown on the site.

Link to the relevant composer.json: 
https://github.com/wdelfuego/nova-wizard/blob/main/composer.json

**Comment by kaiserkiwi** [comment]
I have to update here. I did nothing but now the badge is there. It probably needs some time to process this? It's odd, but my package is listed as Nova 4 package now.

**Comment by RhysLees** [comment]
> I have to update here. I did nothing but now the badge is there. It probably needs some time to process this? It's odd, but my package is listed as Nova 4 package now.

It will take until the next time the package is fetched from packagist i beleive

**Comment by wdelfuego** [comment]
Yep, happened to me too, the Nova 4 support tag was added overnight 👍

**Comment by metasyncSite** [comment]
If  package support both laravel version how label will looks, for example -     "laravel/nova": "^4.0|^5.0"
